concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -485,6 +497,47 @@ jobs:
           params:
             file: generated-bosh-secrets/bosh-secrets.yml
 
+      - task: generate-bosh-ssh-keypair
+        config:
+          platform: linux
+          image_resource:
+            type: docker-image
+            source:
+              repository: governmentpaas/curl-ssl
```

Comment:
Now that this is no longer using `s3init.sh`, There's no need to use this container, we could use a vanilla alpine container, or even a container with openssh already installed (eg our git-ssh container).